docs/get-started/preview_config.json

```diff
@@ -0,0 +1,68 @@
+[
```

Comment:
I'm probably missing it, is this referenced from somewhere in the docs?

Reply:
This is not needed, I will remove